### PR TITLE
Add connector and forwarder examples

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

File: documentation/modules/ref-address-example-forwarders.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+spec:
+  address: myqueue
+  type: queue
+  plan: standard-small-queue
+  forwarders:
+  - name: f1 <1>
+    remoteAddress: remote1/prices/milk <2>
+    direction: in <3>
+----
+<1> (Required) The name of the forwarders. This is used to ensure a unique identity of the forwarder.
```
Comment (jenmalloy) [inline documentation/modules/ref-address-example-forwarders.adoc]:
```suggestion
<1> (Required) Specifies the name of the forwarder, which is used to ensure a unique identity.
```

File: documentation/modules/ref-address-example-forwarders.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+<2> (Required) The remote address to forward messages to. The address must be prefixed with the connector name, and match the address matching patterns defined on the connector.
+<3> (Required) The direction of message flow, either `out` or `in`. `out` will forward messages to the remote endpoint. `in` will forward from the remote endpoint.
```
Comment (jenmalloy) [inline documentation/modules/ref-address-example-forwarders.adoc]:
```suggestion
<3> (Required) Specifies the direction of message flow, which is either `out` or `in`. A value of `out` forwards messages to the remote endpoint. A value of `in` forwards messages from the remote endpoint.
```

File: documentation/modules/ref-address-example-forwarders.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  address: myqueue
+  type: queue
+  plan: standard-small-queue
+  forwarders:
+  - name: f1 <1>
+    remoteAddress: remote1/prices/milk <2>
+    direction: in <3>
+----
+<1> (Required) The name of the forwarders. This is used to ensure a unique identity of the forwarder.
+<2> (Required) The remote address to forward messages to. The address must be prefixed with the connector name, and match the address matching patterns defined on the connector.
```
Comment (jenmalloy) [inline documentation/modules/ref-address-example-forwarders.adoc]:
```suggestion
<2> (Required) Specifies the remote address to forward messages to. The address must be prefixed with the connector name and must be identical to the address matching patterns defined on the connector.
```

File: documentation/modules/ref-address-example-forwarders.adoc [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+// assembly-managing-addresses.adoc
+
+[id='ref-address-example-forwarders-{context}']
+== Address forwarding examples
+
+Forwarders are a way to automatically forward messages from an 'local' address to a remote AMQP address outside of {ProductName}, or forward messages from a remote AMQP address to a 'local' address. Creating a forwarder requires that a connector to the remote AMQP endpoint has been configured for the address space. For more information on connectors, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-address-space-example-connectors-messaging[Address space connector examples].
+
+Forwarding is only supported in the `standard` address space type, and only for the `queue` and `subscription` address type. The `subscription` address type only supports forwarding in the `out` direction.
+
+These examples expect that a connector `remote` has been configured for the address space.
```
[the remark this answers is not in the document]
Comment (lulf) [reply documentation/modules/ref-address-example-forwarders.adoc]: If the procedure links to the connector ref, I think that would be good. The examples are made to 'work together'.

**Comment by lulf** [comment]
@jenmalloy merged your suggestions and squashed

**Pull request merged**